atomix/cluster/src/main/java/io/atomix/raft/cluster/RaftMember.java

```diff
@@ -55,13 +55,6 @@
    */
   void addTypeChangeListener(Consumer<Type> listener);
 
-  /**
-   * Removes a type change listener from the member.
-   *
-   * @param listener The listener to remove from the member.
-   */
-  void removeTypeChangeListener(Consumer<Type> listener);
```

Comment:
Does it mean we never remove listeners? Is this an issue?

Reply:
Personally, I think it is an issue. However, the application seems to run fine without it. I will take a closer look during refactoring of bootstrapping. I want to make sure that for any listener that gets registered at bootstrapping, it is also removed during shutdown.